buf/validate/expression_pb2.pyi

```diff
@@ -0,0 +1,34 @@
+from google.protobuf.internal import containers as _containers
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import message as _message
+from typing import ClassVar as _ClassVar, Iterable as _Iterable, Mapping as _Mapping, Optional as _Optional, Union as _Union
```

Comment:
Sorry for the OT, but I've never seen a stub file with all the imports aliased to have an underscore prefix. I would guess it has something to do with explicitly not re-exporting the name, but afaiu that should not be necessary, so I'm curious to learn if there's another purpose.

Reply:
> I would guess it has something to do with explicitly not re-exporting the name

I would have guessed the same, but also couldn't be sure 🤔. I initially thought `buf generate` used `protoc` under the hood, but after a quick look, I'm not so sure. Regardless, I think the answer will be tucked away in the source of one of those somewhere. Sorry I couldn't be of more help!